tests/test_ftdc_decode.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from aiofile import async_open
+
+from ftdc_tools.ftdc_decoder import FTDC
+
+
+def validateFTDCRecord(record):
```

Comment:
So most of what we're validating is the well-structuredness of the FTDC records? Should we add tests for the case where values are something other than what we expected? e.g. what if `ops` is negative? Can that happen somehow? Can we handle it if so? (Presumably via an error?) Also, what happens if the file in question isn't an FTDC file?

I know you'd mentioned stuff being hard without an encoding lib, so defs push back if there's not a good way to do that kind of testing. And our integration testing of the rollup code might be sufficient.

Reply:
yes, it's hard without encoding lib. Also, IMO the FTDC decoding library should be generic and shouldn't worry about what columns or values they can have. Let me know if you disagree.